docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
+
+## Why use Beetbox?
+
+There are a number of Drupal-focussed Vagrant projects.
+
+* It is leaner than most other projects.
```

Comment:
I totally missed that this is a RTD and not READM.md. sight. ignore all my comments then. The all apply to both this PR and readme.md.